themes/openframeworks/assets/css/style.css

```diff
@@ -419,6 +419,7 @@ a:active {
 	text-decoration: none;
 }
 a.nohover:hover {
+	color: #EE3987;
```

Comment:
The only other element using `nohover` are some `img`s, so adding `color` does not influence those.